state/executor.go

```diff
@@ -104,49 +104,47 @@ func (e *BlockExecutor) CreateBlock(height uint64, lastSignature *types.Signatur
 		maxBytes = int64(e.maxBytes) //nolint:gosec
 	}
 
-	maxGas := state.ConsensusParams.Block.MaxGas
+	// maxGas := state.ConsensusParams.Block.MaxGas
 
-	mempoolTxs := e.mempool.ReapMaxBytesMaxGas(maxBytes, maxGas)
+	// mempoolTxs := e.mempool.ReapMaxBytesMaxGas(maxBytes, maxGas)
 
-	block := &types.Block{
-		SignedHeader: types.SignedHeader{
-			Header: types.Header{
-				Version: types.Version{
-					Block: state.Version.Consensus.Block,
-					App:   state.Version.Consensus.App,
-				},
-				BaseHeader: types.BaseHeader{
-					ChainID: e.chainID,
-					Height:  height,
-					Time:    uint64(time.Now().UnixNano()),
-				},
-				//LastHeaderHash: lastHeaderHash,
-				//LastCommitHash:  lastCommitHash,
-				DataHash:        make(types.Hash, 32),
-				ConsensusHash:   make(types.Hash, 32),
-				AppHash:         state.AppHash,
-				LastResultsHash: state.LastResultsHash,
-				ProposerAddress: e.proposerAddress,
+	header := &types.SignedHeader{
+		Header: types.Header{
+			Version: types.Version{
+				Block: state.Version.Consensus.Block,
+				App:   state.Version.Consensus.App,
 			},
-			Signature: *lastSignature,
-		},
-		Data: types.Data{
-			Txs: toRollkitTxs(mempoolTxs),
-			// IntermediateStateRoots: types.IntermediateStateRoots{RawRootsList: nil},
-			// Note: Temporarily remove Evidence #896
-			// Evidence:               types.EvidenceData{Evidence: nil},
+			BaseHeader: types.BaseHeader{
+				ChainID: e.chainID,
+				Height:  height,
+				Time:    uint64(time.Now().UnixNano()),
+			},
+			//LastHeaderHash: lastHeaderHash,
+			//LastCommitHash:  lastCommitHash,
+			DataHash:        make(types.Hash, 32),
+			ConsensusHash:   make(types.Hash, 32),
+			AppHash:         state.AppHash,
+			LastResultsHash: state.LastResultsHash,
+			ProposerAddress: e.proposerAddress,
 		},
+		Signature: *lastSignature,
+	}
+	data := &types.Data{
+		Txs: toRollkitTxs(txs),
+		// IntermediateStateRoots: types.IntermediateStateRoots{RawRootsList: nil},
+		// Note: Temporarily remove Evidence #896
+		// Evidence:               types.EvidenceData{Evidence: nil},
```

Comment:
?

Reply:
these will be reintroduced as part of fraud proof work as mentioned in the comment.